pkg/config/config.go

```diff
@@ -45,7 +45,9 @@ var (
 	ErrNoConfigStreamlit   = fmt.Errorf("config Streamlit in comfigmap is not found")
 	ErrNoConfigRayClusters = fmt.Errorf("config RayClusters in comfigmap is not found")
 	ErrNoConfigRerank      = fmt.Errorf("config rerankDefaultEndpoint in comfigmap is not found")
+	ErrSystemCliNotFound   = fmt.Errorf("systemCli is not found")
```

Comment:
fmt.Errorf("systemCli is not found") ->  fmt.Errorf("system client is not found")